client/src/components/Navbar/LoggedInNavbar/LoggedInNavbar.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import Tooltip from '@mui/material/Tooltip';
+import MenuItem from '@mui/material/MenuItem';
+import { ListItemIcon, ListItemText, Divider } from '@mui/material';
+import { Person as ProfileIcon, Logout as LogoutIcon } from '@mui/icons-material';
+import { useAuth } from '../../../context/useAuthContext';
+import AuthHeader from '../../../components/AuthHeader/AuthHeader';
+
+const pages = ['My Jobs', 'Massages'];
```

Comment:
Seems a little too simple of a list for routing, love the list of pages to maintain but let's also include some url paths 

```
[ { label: 'My Jobs', to: '/jobs' } ]
```